libethcore/KeyManager.cpp

```diff
@@ -342,6 +342,19 @@ string const& KeyManager::accountName(Address const& _address) const
 	}
 }
 
+void KeyManager::changeName(Address const& _address, std::string const& _name)
+{
+	try
+	{
+		m_keyInfo.at(_address).accountName = _name;
+	}
+	catch (...)
+	{
+		return;
```

Comment:
This function silently does nothing in case of invalid input. Are you sure it is ok in this case? In general it is _very not ok_.